frontend/src/apis/solutions.ts

```diff
@@ -17,8 +17,11 @@ interface GetSolutionSummariesResponse {
   data: SolutionSummary[];
 }
 
-export const getSolutionSummaries = async (): Promise<SolutionSummary[]> => {
-  const { data } = await develupAPIClient.get<GetSolutionSummariesResponse>(PATH.solutionSummaries);
+export const getSolutionSummaries = async (filter: string = 'all'): Promise<SolutionSummary[]> => {
```

Comment:
```suggestion
export const getSolutionSummaries = async (filter: string = HASHTAGS.all): Promise<SolutionSummary[]> => {
```

여기도 만들어뒀던 상수 이용할 수 있을 것 같아요!